spec/factories/rounds.rb

```diff
@@ -19,6 +28,7 @@
     members_can_propose_buckets true
```

Comment:
if a round is open for contributions, shouldn't `members_can_propose_buckets` be false?